erpnext/buying/doctype/request_for_quotation/request_for_quotation.js

```diff
@@ -56,21 +56,48 @@ frappe.ui.form.on("Request for Quotation",{
 
 	get_suppliers_button: function (frm) {
 		var doc = frm.doc;
+		console.log('s')
 		var dialog = new frappe.ui.Dialog({
 			title: __("Get Suppliers"),
 			fields: [
-				{	"fieldtype": "Select", "label": __("Get Suppliers By"),
+				{
+					"fieldtype": "Select", "label": __("Get Suppliers By"),
 					"fieldname": "search_type",
-					"options": "Tag\nSupplier Group", "reqd": 1 },
-				{	"fieldtype": "Link", "label": __("Supplier Group"),
+					"options": "Tag\nSupplier Group", "reqd": 1,
+					'onchange': function() {
+						if(dialog.fields_dict.search_type.value == 'Tag'){
+							frappe.call({
+								method: 'erpnext.buying.doctype.request_for_quotation.request_for_quotation.get_supplier_tag',
+								async: false,
+							}).then(r => {
+								var t =[]
+								for(var tag in r.message){
+									var arr = r.message[tag][0].split(",").filter(t =>Boolean(t))
+									t = t.concat(arr)
+								}
+								t = [...new Set(t)];
+								dialog.set_df_property("tag", "options", t)
+						});
+						}
+					}
+				},
+				{
+					"fieldtype": "Link", "label": __("Supplier Group"),
 					"fieldname": "supplier_group",
-					"options": "Supplier Group",	"reqd": 0,
-					"depends_on": "eval:doc.search_type == 'Supplier Group'"},
-				{	"fieldtype": "Data", "label": __("Tag"),
-					"fieldname": "tag",	"reqd": 0,
-					"depends_on": "eval:doc.search_type == 'Tag'" },
-				{	"fieldtype": "Button", "label": __("Add All Suppliers"),
-					"fieldname": "add_suppliers", "cssClass": "btn-primary"},
+					"options": "Supplier Group",
+					"reqd": 0,
+					"depends_on": "eval:doc.search_type == 'Supplier Group'"
+				},
+				{
+					"fieldtype": "Select", "label": __("Tag"),
+					"fieldname": "tag",
+					"reqd": 0,
+					"depends_on": "eval:doc.search_type == 'Tag'",
+				},
+				{
+					"fieldtype": "Button", "label": __("Add All Suppliers"),
+					"fieldname": "add_suppliers", "cssClass": "btn-primary"
```

Comment:
`cssClass` property doesn't do anything

```suggestion
					"fieldname": "add_suppliers",
```

Reply:
done